addOns/help/src/main/javahelp/contents/start/proxies.html

```diff
@@ -30,8 +30,8 @@ <H3>Firefox (on Windows)</H3>
 <table>
 <tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>Select the</td><td> 'Tools' menu </td></tr>
 <tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>Select the</td><td> 'Options..' menu item</td></tr>
-<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>Press the</td><td> 'Advanced' button</td></tr>
-<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>Select the</td><td> 'Network' tab</td></tr>
+<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>Select the</td><td> 'General' panel</td></tr>
+<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td>Scroll to</td><td> 'Network Settings' section</td></tr>
```

Comment:
Win specific (but likely others as well, since it's Firefox). Should this include setting "Use t his proxy server for all protocols"?
![image](https://user-images.githubusercontent.com/7570458/60202464-9cdee080-9818-11e9-9657-09076fe73e55.png)

Do we want to mention this about:config setting for proxying local traffic? https://security.stackexchange.com/a/212487/142657

Reply:
Tweaked.